content/en/blog/_posts/2018-10-11-topology-aware-volume-provisioning.md

```diff
@@ -139,8 +139,7 @@ logs-web-1      us-central1-a
 
 ## How can I learn more?
 
-Official documentation on the topology-aware dynamic provisioning feature is available here:
-/docs/concepts/storage/storage-classes/#volume-binding-mode
+Official documentation on the topology-aware dynamic provisioning feature is available [here](https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode)
```

Comment:
Wherever possible, we would like to use the link without the k8s.io prefix. In this case it is better to remove the absolute URL.

```suggestion
Official documentation on the topology-aware dynamic provisioning feature is available [here](/docs/concepts/storage/storage-classes/#volume-binding-mode)
```